### DISCUSSION TITLE
Volume toast not appearing for Increase/Decrease Volume for all devices

### DISCUSSION THREAD

**Discussion by XangelMusic** [File-New-Project/EarTrumpet#1453] [Q&A]
Normally in Windows whenever you change the volume, it will display a toast/pop-up of the volume in the top left corner of the screen. But after assigning shortcuts to change volume for all devices, the toast is no longer visible.

How can I see the volume pop-up again? Its useful to show which device I'm currently using.

**Comment by riverar** [comment]
Are you referring to one of these?

![image](https://github.com/File-New-Project/EarTrumpet/assets/475132/dddf257d-9a15-491a-8987-426b8a186991)

![image](https://github.com/File-New-Project/EarTrumpet/assets/475132/001e255a-aeb3-4068-9977-3e97c7568d89)

**Reply by XangelMusic** [reply]
Yes I am, mainly the first image. That doesn't show anymore when assigning the volume change shortcuts.
